src/app/Plans/types.ts

```diff
@@ -1,81 +1,48 @@
-// TODO This is speculative until details are avail.
-
-import { IHook } from '@app/Hooks/types';
-import { IVMwareProvider, ICNVProvider } from '@app/Providers/types';
+import { Hook } from '@app/Hooks/types';
+import { CR } from '@app/common/types';
+import { Mapping } from '@app/Mappings/types';
+import { IVMwareProvider, ICNVProvider, IHost } from '@app/Providers/types';
 
 export interface IAddPlanDisabledObjModel {
   isAddPlanDisabled: boolean;
   disabledText: string;
 }
 
-export interface IMigHooks {
-  before: boolean;
-  beforeHook: IHook;
-  after: boolean;
-  afterHook: IHook;
+export interface PlanVM {
```

Comment:
Just for consistency and to distinguish between components and interfaces in imports, let's put an `I` prefix on all our interfaces.

Honestly I wonder if we should be putting `E` in front of enums and `T` or `U` in front of union types. I sometimes have to remember what is and isn't a component when I look at imports. I was guilty of that here with `Mapping`.

Reply:
+1 I agree with the I in front of interfaces. Keeps things simple & allows for quick identification of TS specific pieces.

Reply:
@mturley, @ibolton336, yeah I guess I got excited with the `Mapping` ;)
I'm fine with using `I` for interface so we're consistent.
For `E`and others, I'll tend to not using it unless it really help when reading the code. But if the TS trend is going that I won't be able to resist it.